frontend/app/stripe/refreshAccountLink/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+import { useUser } from "@/components/utility/UserContext";
+import { getStripeStandardAccounLink } from "@/services/src/stripeService";
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
+export default function RefreshAccountLink() {
+  const { user } = useUser();
+  const router = useRouter();
+  useEffect(() => {
+    getStripeStandardAccounLink(user.userId, "http://localhost:3000/organiser").then((link) => {
```

Comment:
will need to change localhost:3000 to dynamic routing when deployed

Reply:
jks i lied, you stripe needs a real live FULL url, so we probably will need to make these environment dependent

Reply:
i can probably do it dynamically using `window.location.hostname`... im just thinking of our preview environment where the host name is uniquely generated per deployment